factory.py

```diff
@@ -39,7 +39,7 @@ def create_from_verification_data(verification_data: VerificationData) -> arshin
             verification.applicable.sign_pass = verification.applicable.sign_mi = False
 
         verification.metrologist = verification_data.metrologist
-        verification.means = MeansFactory.create_test(verification_data.test_dev_num)
+        verification.means = MeansFactory.create_test(verification_data.test_dev_num, config.company_config['test_devices'])
```

Comment:
I suggest you to at least create an issue to add data validation in the future.

You're operating with non-validated data more and more with each commit.